src/V3RandomizeMethod.h

```diff
@@ -0,0 +1,32 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// DESCRIPTION: Verilator: Node attributes/ expression widths
+//
+// Code available from: https://verilator.org
+//
+//*************************************************************************
+//
+// Copyright 2003-2020 by Wilson Snyder. This program is free software; you
+// can redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#ifndef _V3RANDOMIZE_METHOD_H_
+#define _V3RANDOMIZE_METHOD_H_ 1
+
+#include "config_build.h"
+#include "verilatedos.h"
+
+#include "V3Ast.h"
+
+class V3RandomizeMethod {
```

Comment:
```suggestion
class V3RandomizeMethod final {
```

Add "final" to all classes - recent master change that gave perf boost.